pilot/pkg/proxy/envoy/v1/mixer.go

```diff
@@ -38,21 +40,45 @@ const (
 	// MixerFilter name and its attributes
 	MixerFilter = "mixer"
 
+	// AttrSourcePrefix all source attributes start with this prefix
+	AttrSourcePrefix = "source"
+
 	// AttrSourceIP is client source IP
 	AttrSourceIP = "source.ip"
 
 	// AttrSourceUID is platform-specific unique identifier for the client instance of the source service
 	AttrSourceUID = "source.uid"
 
+	// AttrDestinationPrefix all destination attributes start with this prefix
+	AttrDestinationPrefix = "destination"
+
 	// AttrDestinationIP is the server source IP
 	AttrDestinationIP = "destination.ip"
 
 	// AttrDestinationUID is platform-specific unique identifier for the server instance of the target service
 	AttrDestinationUID = "destination.uid"
 
+	// AttrDestinationLabels is Labels associated with the destination
+	AttrDestinationLabels = "destination.labels"
+
 	// AttrDestinationService is name of the target service
 	AttrDestinationService = "destination.service"
 
+	// AttrIP represents IP address
```

Comment:
These should probably have "suffix" in the name since they aren't valid attribute names on their own, e.g. `AttrIPSuffix`